app/json/slate_configs.json

```diff
@@ -221,6 +221,7 @@
     "id": "d9c6ddfa-a58d-402d-9664-4190ba197ea6",
     "displayName": "Popular with Pocket readers  Stories from across the web",
     "description": "Algorithmic Business Slate",
+    "curatorTopicLabel": "Business",
```

Comment:
the reason I didn't have this to start was that the curator labels are manual for the curated slates, but these are all estimates.  it's possible that we might want to make that distinction explicit with something like `estimatedTopicLabel` for the field, but I don't feel strongly.

Reply:
Understandable. Functionally we are using it to just match slates to predicted topic labels for a user vector. I think at this point the distinction is more that `curatorTopicLabels` are a specific ontology rather than saying curators choose these labels, so I am ok with keeping the label the same. I could see us having `userTagTopicLabel` in the future which is the ontology based on user tags.